restapi/src/test/java/io/stargate/web/docsapi/service/QueryExecutorTest.java

```diff
@@ -227,39 +257,42 @@ void testFullScanFinalPagingState(int pageSize) throws Exception {
     // paging state in the page that contains the last row to emulate C* behaviour, which cannot
     // say for sure that there are no more rows available in this case. Therefore, in that case
     // the last doc will have a non-null paging state, but using it will yield no other docs.
-    boolean shouldHavePagingState = (8 % pageSize) == 0;
-    withFiveTestDocs(pageSize);
+    int effectivePageSize = Math.max(10, pageSize);
```

Comment:
Again, should we start pageSize at 10 since that's the smallest possible pageSize?